index.html

```diff
@@ -189,7 +196,21 @@ <h3>Skills</h3>
                 </div>
             </div>
         </section>
-    </main>
+    </section>
+
+    <section class="contact-section">
+        <h1 id="contact-card-title">Contact</h1>
+
+        <p>If you have an application you are interested in developing, a feature that you need
+            built or a project that needs coding. I’d love to help with it </p>
+
+        <form action="#" method="POST" class="contact-form"></form>
+        <input type="text" name="_name" placeholder="Name" class="form-field" required>
+        <input type="email" name="_replyto" placeholder="Email" class="form-field" required>
+        <textarea name="_message" placeholder="Write your message here" maxlength="500"
+            class="form-field contact-message"></textarea>
+        <button type="submit" id="contact-btn">Get In Touch</button>
```

Comment:
- [ ] Your form is supposed to submit valid data successfully. However, this is not the case with your form. Please work on that issue.

Reply:
- [ ] Please ensure you follow the design specified on Figma for the button.